cppcache/src/ThreadPool.cpp

```diff
@@ -16,6 +16,8 @@
  */
 #include "ThreadPool.hpp"
 
+#include <sstream>
```

Comment:
I think this might be a remaining of some dev debugging. Is this needed anymore?

Reply:
You're right. Not needed.